sys/kern/subr_turnstile.c

```diff
@@ -1129,10 +1129,9 @@ DB_SHOW_COMMAND(turnstile, db_show_turnstile)
 	 * First, see if there is an active turnstile for the lock indicated
 	 * by the address.
 	 */
-	lock = (struct lock_object *)addr;
-	tc = TC_LOOKUP(lock);
+	tc = TC_LOOKUP(addr);
```

Comment:
Weird that this one's done differently to sleepqueue

Reply:
I think it happened accidentally over time, perhaps we should clean it up upstream (also @bsdjhb)? Does not seem to be a specific reason for the difference.